combat-trainer.lic

```diff
@@ -1055,6 +1094,36 @@ class SpellProcess
     game_state.casting = false
   end
 
+  def check_regalia(game_state)
+    game_state.swap_regalia_type = nil if game_state.swap_regalia_type == game_state.last_regalia_type
+    return if game_state.casting
+    return unless game_state.swap_regalia_type
+    if @regalia_spell.nil?
+      echo "Couldn't find a regalia waggle.  Using default values for the spell."
+      @regalia_spell = get_data('spells').spell_data['Regalia']
+    end
+    
+    case game_state.swap_regalia_type
+    when 'Light Armor'
```

Comment:
💡 This would make another good error-check (whether in this Process or in `validate.lic`)

Reply:
Added a check to validate to throw an error if it contains a non-armor-skill.